src/mol-util/param-definition.ts

```diff
@@ -77,7 +77,8 @@ export namespace ParamDefinition {
         type: 'select'
         /** array of (value, label) tuples */
         options: readonly (readonly [T, string] | readonly [T, string, string | undefined])[]
-        cycle?: boolean
+        cycle?: boolean,
+        _optionSet?: Set<T>
```

Comment:
Is this worthwhile? how many options need there be for this to be needed? how often normalized?

It should be excluded in PD.clone

Reply:
You're right, probably not worth it at this point.